packages/api/core/test/fast/publish_spec.js

```diff
@@ -203,25 +217,33 @@ describe('publish', () => {
 
       it('should successfully restore values and pass them to publisher', () => {
         expect(makeStub.callCount).to.equal(0);
-        expect(publisher.callCount).to.equal(2, 'should call once for each platform (make run)');
-        const darwinIndex = publisher.firstCall.args[0].platform === 'darwin' ? 0 : 1;
+        expect(publisherSpy.callCount).to.equal(2, 'should call once for each platform (make run)');
+        const darwinIndex = publisherSpy.firstCall.args[0].platform === 'darwin' ? 0 : 1;
         const win32Index = darwinIndex === 0 ? 1 : 0;
-        const darwinArgs = publisher.getCall(darwinIndex).args[0];
-        expect(darwinArgs.artifacts.sort()).to.deep.equal(
+        const darwinArgs = publisherSpy.getCall(darwinIndex).args[0];
+        const darwinArtifacts = [];
+        for (const result of darwinArgs.makeResults) {
+          darwinArtifacts.push(...result.artifacts);
+        }
+        expect(darwinArtifacts.sort()).to.deep.equal(
           fakeMake('darwin').reduce((accum, val) => accum.concat(val.artifacts), []).sort()
         );
         expect(darwinArgs.packageJSON).to.deep.equal({ state: 1 });
         expect(darwinArgs.authToken).to.equal(undefined);
-        expect(darwinArgs.tag).to.equal(null);
+        expect(darwinArgs.tag).to.equal('1.0.0');
```

Comment:
@MarshallOfSound I'm not sure about this change I made to get the publisher tests working, let me know if this is incorrect.